docs/error-codes.md

```diff
@@ -41,13 +41,14 @@ Here is a summary of all the error codes thrown by the contract.
 
 | Error Code       | Error Label      | Description                                           |
 |------------------|------------------|-------------------------------------------------------|
-| 300 | `internal_tick_not_exist_err` | Tick is not initialized. |
-| 301 | `internal_epoch_bigger_than_now_err` | Time now is smaller than epoch time. |
-| 302 | `internal_fee_more_than_100_percent_err` | The `const_fee_bps` is initialized to be higher than 10000 (100%). |
-| 303 | `internal_303` | Thrown when `(p.s.sqrt_price - sqrt_price_new)` is not nat. |
-| 304 | `internal_304` | Thrown when `(sqrt_price_new - p.s.sqrt_price)` is not nat. |
-| 305 | `flip_fee_growth_outside_err` | Flip for `fee_growth_outside` failed. (This is an invariant of the contract). |
-| 306 | `internal_306` | Thrown when `(p.dx - dx_consummed)` is not nat. |
+| 300 | `internal_impossible_err` | Generic impossible error. |
+| 301 | `internal_tick_not_exist_err` | Tick is not initialized. |
+| 302 | `internal_epoch_bigger_than_now_err` | Time now is smaller than epoch time. |
+| 303 | `internal_fee_more_than_100_percent_err` | The `const_fee_bps` is initialized to be higher than 10000 (100%). |
+| 304 | `internal_303` | Thrown when `(p.s.sqrt_price - sqrt_price_new)` is not nat. |
```

Comment:
Since we put the the impossible error on the top, the error code and label now are different. Same with `internal_304` and `internal_306`.

We should make them the same. (If we have a label for them, it would be much better, but I guess maybe later on).

Reply:
Yeah, fair.

Reply:
Looks like the CI failed, we have to rename the label in the contract usage as well.

Reply:
Oh, right. Fixed now